engines/mohawk/myst_state.h

```diff
@@ -86,6 +86,15 @@ enum ActiveAge {
 	kAchenarEnding = 10
 };
 
+// Various states that Atrus can be in when in Dini
+enum DiniEnding {
```

Comment:
Typo.  The location is called D'ni, not Dini.  `DniEnding` would be sufficiently readable.